### PR TITLE
fix tailwind compile time

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

- tailwind was trying to purge thousands of node module files

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: studio/tailwind.config.js [hunk 1]
```diff
@@ -2,20 +2,12 @@ const config = require('config/tailwind.config')
 
 module.exports = config({
   content: [
-    './../packages/common/**/*.{ts,tsx}',
```
Comment (MildTomato) [inline studio/tailwind.config.js:5]: we don't use this anyway

File: studio/tailwind.config.js [hunk 2]
```diff
@@ -2,20 +2,12 @@ const config = require('config/tailwind.config')
...
-    './../packages/ui/**/*.{tsx,ts,js}',
+    './../packages/ui/src/**/*.{tsx,ts,js}',
     './pages/**/*.{js,ts,jsx,tsx}',
     './components/**/*.{js,ts,jsx,tsx}',
-    './lib/**/*.{js,ts,jsx,tsx}',
-    './lib/**/**/*.{js,ts,jsx,tsx}',
-    './../packages/ui/src/lib/theme/defaultTheme.ts',
```
Comment (MildTomato) [inline studio/tailwind.config.js:11]: covered in `./../packages/ui/**/*.{tsx,ts,js}',

File: studio/tailwind.config.js [hunk 3]
```diff
@@ -2,20 +2,12 @@ const config = require('config/tailwind.config')
...
-    './../packages/ui/src/components/**/*.{ts,tsx}',
```
Comment (MildTomato) [inline studio/tailwind.config.js:12]: covered in `./../packages/ui/**/*.{tsx,ts,js}'`

File: studio/tailwind.config.js [hunk 4]
```diff
@@ -2,20 +2,12 @@ const config = require('config/tailwind.config')
...
-    // purge styles from supabase ui theme
-    // './../node_modules/ui/dist/config/default-theme.js',
     // purge styles from grid library
     './../node_modules/@supabase/grid/src/components/**/*.{js,ts,jsx,tsx}',
     './../node_modules/@supabase/grid/src/components/**/**/*.{js,ts,jsx,tsx}',
-    './internals/**/*.{js,ts,jsx,tsx}',
```
Comment (MildTomato) [inline studio/tailwind.config.js:18]: don't need

**Comment by MildTomato** [comment]
comments above in case anyone wondering why im removing file paths

**Review by alaister — APPROVED** [review]

**Pull request merged**